smartcar/types.py

```diff
@@ -266,6 +266,31 @@ def format_capabilities(capabilities_list: List[dict]) -> List[Capability]:
 
 Response = NamedTuple("Response", [("body", dict), ("meta", namedtuple)])
 
+# ===========================================
+# Lock Status Tuples
+# ===========================================
+
+Door = NamedTuple("Door", [("type", str), ("status", str)])
+Window = NamedTuple("Window", [("type", str), ("status", str)])
+Sunroof = NamedTuple("Sunroof", [("type", str), ("status", str)])
+Storage = NamedTuple("Storage", [("type", str), ("status", str)])
+ChargingPort = NamedTuple("ChargingPort", [("type", str), ("status", str)])
+
+
+LockStatus = NamedTuple(
+    "LockStatus",
+    [
+        ("is_locked", bool),
+        ("doors", List[Door]),
+        ("windows", List[Window]),
+        ("sunroof", List[Sunroof]),
+        ("storage", List[Storage]),
+        ("charging_port", List[ChargingPort]),
```

Comment:
Good, this is singular. I forgot that singular is the name we use for the external users